Xamarin.Forms.Platform.Android/AppCompat/TabbedPageRenderer.cs

```diff
@@ -633,6 +640,42 @@ void UpdateBottomNavigationViewIcons()
 			}
 		}
 
+		void UpdateBottomNavigationShiftMode()
+		{
+			if (IsDisposed || _bottomNavigationView == null || !IsBottomTabPlacement)
+				return;
+			var bottomNavigationView = _bottomNavigationView;
+
+			try
+			{
+				var menuView = bottomNavigationView.GetChildAt(0) as BottomNavigationMenuView;
+				if (menuView == null)
+					return;
+
+				var shiftMode = menuView.Class.GetDeclaredField("mShiftingMode");
+				shiftMode.Accessible = true;
```

Comment:
I do we set true, and then False?

Reply:
It's just the whacky way you have to do it
https://stackoverflow.com/questions/40176244/how-to-disable-bottomnavigationview-shift-mode